components/dashboard/src/components/RepositoryFinder.tsx

```diff
@@ -156,16 +207,15 @@ export default function RepositoryFinder({
                 result.push({
                     id: "bitbucket-server",
                     element: (
-                        <div className="text-sm text-pk-content-tertiary">
-                            <div className="flex items-center">
-                                <Exclamation2 className="w-4 h-4"></Exclamation2>
-                                <span className="ml-2">Bitbucket Server only supports searching by prefix.</span>
-                            </div>
+                        <div className="text-sm text-pk-content-tertiary flex items-center">
+                            <Exclamation2 className="w-4 h-4 mr-2" />
+                            <span>Bitbucket Server only supports searching by prefix.</span>
                         </div>
                     ),
                     isSelectable: false,
-                } as ComboboxElement);
+                });
             }
+
             if (searchString.length < 3) {
```

Comment:
We can remove `Please type at least 3 characters to search.` suggestion while showing demo repos only.

```suggestion
            if (searchString.length < 3 && !showExamples) {
```

Reply:
we should maybe only use the searchQuery to decide what to show below three carachters and afterward do the regular search. This would also better work with the placeholder text.

Reply:
> we should maybe only use the searchQuery to decide what to show below three carachters and afterward do the regular search. This would also better work with the placeholder text.

I will keep this change as out of scope of this PR